### PR TITLE
feat : added product page

### PR DESCRIPTION
**Pull request by ArjunJunna**
Have added the product page.

### CODE REVIEW

File: css/layout/content.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+.card__container .card-details {
+  padding: 0.6rem;
+}
+
+.card-details .detail-heading {
+  font-size: 1.5rem;
+  margin: 1rem 0;
+  font-family: 'Roboto Mono', monospace;
```
Comment (MayankKumar10) [inline css/layout/content.css]: if you can define font family in html if you use one font family in the entire project

**Review by MayankKumar10 — APPROVED** [review]

**Review by MayankKumar10 — APPROVED** [review]
Everything else LGTM you can merge it

File: css/layout/content.css (hunk 1 above)
Comment (ArjunJunna) [reply css/layout/content.css]: This helped a lot. Thank you.

File: css/layout/footer.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  padding: 3rem;
+  display: flex;
+  flex-direction: column;
+  
+}
+
+.list {
+  
+  padding: 3rem;
+}
```
Comment (BalajiHarinath) [inline css/layout/footer.css]: remove the empty lines

File: css/layout/sidebar.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+.sidebar__container .sidebar-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.sidebar-h5 {
+
```
Comment (BalajiHarinath) [inline css/layout/sidebar.css]: remove the empty line here as well

File: css/layout/sidebar.css [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+.slider:hover {
+  opacity: 1;
+}
+
+.slider::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 15px;
+  height: 15px;
```
Comment (BalajiHarinath) [inline css/layout/sidebar.css]: convert these to rem.

File: css/layout/sidebar.css [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+  border-radius: 50%;
+  background: var(--primary-color);
+  cursor: pointer;
+}
+
+.slider::-moz-range-thumb {
+  width: 15px;
+  height: 15px;
```
Comment (BalajiHarinath) [inline css/layout/sidebar.css]: convert to rem

**Review by BalajiHarinath — APPROVED** [review]
There are a few empty lines, clean them up and also consistently maintain a line gap after each css selector. Apart from these everything looks good to me.

File: css/layout/footer.css (hunk 2 above)
Comment (ArjunJunna) [reply css/layout/footer.css]: yeah sometimes prettier and auto formatter won't work... Will take care of that thank you.

File: css/layout/sidebar.css (hunk 3 above)
Comment (ArjunJunna) [reply css/layout/sidebar.css]: done

File: css/layout/sidebar.css (hunk 4 above)
Comment (ArjunJunna) [reply css/layout/sidebar.css]: sure thank you

**Pull request merged**